### ISSUE TITLE
Where are the doc links for triggers?

### ISSUE THREAD

**Issue by chrimc62** [microsoft/BotFramework-Composer#7226]
## Describe the bug

Triggers have no doc link.

## Version

Version:  1.4.0-nightly.237625.d1378c6
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

## To Reproduce

1. Create a trigger

## Expected behavior

There should be doc links for every composer action or trigger.

## Screenshots
![image](https://user-images.githubusercontent.com/17637681/115599368-634e5380-a290-11eb-80ec-be694a407df0.png)

**Comment by cwhitten** [comment]
@chrimc62 see the tooltip on the Trigger phrases element.

cc @sangwoohaan for comment

![Screen Shot 2021-04-22 at 9 44 59 AM](https://user-images.githubusercontent.com/1156704/115753040-91df3380-a34f-11eb-8246-6f6369d665fb.png)

**Comment by sangwoohaan** [comment]
Yeah we should definitely add Learn more link to the Intent trigger page on the top label field like this
![Screen Shot 2021-04-22 at 11 15 51 AM](https://user-images.githubusercontent.com/50963385/115765877-23ed3900-a35c-11eb-974a-dc54437dd406.png)
All the triggers are missing links to relevant documentations actually – I'll collect them and post here soon.

**Comment by sangwoohaan** [comment]
Here's the list of triggers and associated documentations :
https://microsoft-my.sharepoint-df.com/:x:/g/personal/sahan_microsoft_com1/EVGYu5w-FrpBsGoHWcZcTXQBbepwYVUMc7I13YornmTbTA?e=VBhym2

Let's add Learn more buttons to Triggers, and link to these docs.

**Comment by mareekuh** [comment]
Please follow the guidelines outlined in this spec:
https://microsoft.sharepoint.com/:w:/r/teams/ConversationalAI785/_layouts/15/Doc.aspx?sourcedoc=%7BC52BBE6D-42E3-4B35-BC6F-B94704C047BF%7D&file=R11%20Spec%20-%20Tooltips%20and%20Help%20Links.docx&action=default&mobileredirect=true

When links to documentation is needed, add instructional copy in the modal, page or properties panel where the concept or label is introduced. If a tooltip needs a 'learn more' link, this means that the/a tooltip is not sufficient and either needs to contain more informative copy, or the instruction needs to be elevated and visible at all times vs in a tooltip.

< copy pasted from the document above >


Content type | Source | Guidelines
-- | -- | --
Tooltip | SDK / Code comment | Provide the user with information about a UX element or label (example: Hover over highlighter icon, tooltip shows label: Highlighter') Do not replace, nor should be used as instructional copy. Tooltip is short: Less than 2 short sentences in English. Text in the tooltip shall not repeat the label or name of the element it is intended to describe or the watermark text inside an entry field/edit box. Meets accessibility guidelines (including how they are invoked and presented) Meets Fluent design guidelines May include visuals or diagrams Cannot be clicked


Help link | UI schema/Documentation | Help links should be contextual to the user experience. A “learn more” link is usually placed at the end of a string or a UI card or as curated collection of documentation links on a blade. Context sensitive – should point to the specific section in documentation that pertains to the context/topic of the help link. Link to the specific section of an article if necessary Determine the best link text to use, considering accessibility requirements Where possible, the help link is specific about the content it will link to (vs ‘Learn more’) Create a fwlink and reference it in code

Instructional copy | UI schema | Instructional text provides the user with information about what to DO/how to use the UX. (not: provide information about the product) Instructional text is created with specific context in mind to best help the user with the specific task at hand. Instructional text shall not repeat the label, name of the element it is intended to describe or the watermark text inside an entry field/edit box. Where more information is needed than 1 or 2 sentences (in English) a link for more information is included. This link must go to the specific section or paragraph (vs the top of an article that covers multiple topics) Meet accessibility guidelines (including how they are invoked and presented) Meet Fluent design guidelines Instructional text may be accompanied  by illustrations or diagrams, below or next to the copy.


Watermark copy | UI schema | Text that lives inside an edit box/entry field Single word or short sentence to show an example of what information is expected to be filled out in this box,

Error messages | Composer code |
